### ISSUE TITLE
[Bug]: Caused by: java.nio.file.FileSystemException: .: Operation not permitted

### ISSUE THREAD

**Issue by soulstriker23** [parkervcp/eggs#1308]
### Panel Version

1.5.1

### Wings Version

1.4.7

### Service

minecraft:Forge Enhanced

### Modified

No, I did not modify the egg

### Expected Behavior

Updating the panel and using the new images of **ghcr.io/pterodactyl/yolks:java_8**
Installed forge 1.16.5, I expect it to start up without error after accepting the EULA.

### Actual Behavior

Server failed to start with this error: `[00:52:38] [main/FATAL] [minecraft/Main]: Failed to start the minecraft server
java.lang.RuntimeException: java.nio.file.FileSystemException: .: Operation not permitted`

![image](https://user-images.githubusercontent.com/25358676/130300886-7a20158d-c580-409a-aaf4-8f9b5707a014.png)

### Steps To Reproduce

When using the latest egg images of **ghcr.io/pterodactyl/yolks:java_8** Install basic forge enhanced egg using Minecraft version 1.16.5, then start the server. 

The error will be displayed after starting up.

**Note**
This does not seem to happen when using the previous images such as:
**quay.io/pterodactyl/core:java**

**Comment by Software-Noob** [comment]
Can you try again and reproduce this issue?

**Comment by soulstriker23** [comment]
> Can you try again and reproduce this issue?

Yep, redid it and still the same issue. Video link: https://streamable.com/jueaz1

**Comment by Software-Noob** [comment]
This is resolved with https://github.com/pterodactyl/yolks/pull/14
